tests/export_test_scenes.py

```diff
@@ -24,14 +28,22 @@ def main():
             out_path, blend_name = os.path.split(full_path)
             out_path = os.path.join(
                 out_path,
-                '../godot_project/exports/',
+                '../../godot_project/exports/',
                 blend_name.replace('.blend', '.escn')
                 )
             print(out_path)
-            export_escn(out_path)
+            export_escn(out_path, config)
             print("Exported")
 
 
+def main():
+    test_scenes_dir = os.path.join(os.getcwd(), "tests/test_scenes")
+    for dir_name in os.listdir(test_scenes_dir):
```

Comment:
Only one folder deep. Hmm. Maybe use os.walk, and pass the directory to export_blend (as per above, so preserve the folder structure between the two places).